job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java

```diff
@@ -250,10 +250,14 @@ private synchronized void updateStatus() {
           mPlanInfo.setStatus(Status.FAILED);
           if (mPlanInfo.getErrorMessage().isEmpty()) {
             mPlanInfo.setErrorMessage("Task execution failed: " + info.getErrorMessage());
+            LOG.info("Job failed Id={} Error={}", mPlanInfo.getId(), info.getErrorMessage());
```

Comment:
should this be `LOG.error`? Also, is there something in `TaskInfo info` that would be useful to print to the log?

Reply:
There is a LOG.info of when the job starts that contains more details. Job failing is actually fairly frequent enough that info log is okay.